packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -258,6 +259,9 @@ export function createOdspNetworkError(
             break;
     }
 
+    if (innerMostErrorCode === "fluidInvalidSchema") {
```

Comment:
Should this be scoped to a particular error code? Or maybe assert that it is used only with specific error code? Hard to do it here, as any assert will result in some wrong error, but I wonder how we should ensure service is correct? I.e. it does not give us conflicting messages (let's say 507 (out of storage) with "fluidInvalidSchema" mark-up).

Reply:
how about checking if the error type is also generic. from Kusto it looks like the fluidInvalidSchema errors are all status 400